### DISCUSSION TITLE
Timeout should be moved to a tab

### DISCUSSION THREAD

**Discussion by ericnewcomer** [nyaruka/floweditor#1141] [Ideas]
Now that we have tabs, timeout should be treated as a more advanced feature instead of placing it  in the dialog gutter.

**Comment by ericnewcomer** [comment]
Not entirely sure I agree with this actually. While a tab might be more consistent this kind of feels like "the last rule" to an extent. If we want it out of the gutter maybe it belongs just under the rule list.

**Comment by rowanseymour** [comment]
Is this the only place we put things in the gutter?

**Comment by ericnewcomer** [comment]
Ya, we don't currently use the gutter for anything else
